.rubocop.yml

```diff
@@ -2,6 +2,8 @@ inherit_from:
   - .rubocop_todo.yml
   - .rubocop_dnsimple.yml
 
+require: rubocop-performance
```

Comment:
Is this change required by the upgrade, or was is a separate task?

Reply:
Not required of this upgrade, but was trying to keep things in sync. I can wipe it out and open a new PR for it.

Reply:
OK. Fine to leave it here. Next time I suggest to either mention it in the review comment, or feel free to actually push it straight to master.